types/material-ui/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for material-ui v0.16.0
+// Type definitions for material-ui v0.16.1
```

Comment:
Right so this is a great question and I may have the wrong information - I was under the impression that the typings version is not strictly related to the library version of the thing they type. Even for very popular libraries, I may be using a version like 12 (or some high number) but the typings version is 1.2 (or some low number) - so I thought they were independent and that's why I just upped it to x.1. If I am wrong, I am happy to update it to 0.17.2.